examples/vue/env.ts

```diff
@@ -0,0 +1,10 @@
+import { h } from 'vue';
+
+// Parcel picks the `source` field of the monorepo packages and thus doesn't
+// apply the Babel config. We therefore need to manually override the constants
+// in the app, as well as the React pragmas.
+// See https://twitter.com/devongovett/status/1134231234605830144
+(global as any).__DEV__ = process.env.NODE_ENV !== 'production';
+(global as any).__TEST__ = false;
+(global as any).h = h;
+(global as any).createElement = h;
```

Comment:
is both h and createElement needed?

Reply:
It depends on the dependencies of the sandbox. It seems like it works without `h` and `createElement` here now. So I'll remove them.

I plan to re-iterate on all `env.ts` files once we introduce #499, because this PR will introduce new rendering deps (from Parcel's POV). The goal would be that `env.ts` supports all deps that we might introduce in the sandbox, even though we don't right now, to avoid any trouble later on.

Reply:
See 43d10ce5eb0ec9252bc7664940729df749a14e9a.